shell.nix

```diff
@@ -20,6 +20,7 @@ mkShell {
     cargo-expand
     cargo-udeps
     git
+    nodejs
```

Comment:
where's nodejs required?

Reply:
https://github.com/openebs/mayastor-extensions/blob/13a77d12e2d0224ef457b1c86676721c4fd6b155/.pre-commit-config.yaml#L30
I've actually changed it now to install commitlint directly into the nix-shell, it'll still use nodejs because it's written for nodejs but it becomes a bit more explicit I think, what do you think?

Reply:
Thanks, this is good if the commitlint's version matches our requirements.